cli/src/pcluster/config/cluster_config.py

```diff
@@ -1022,12 +1016,20 @@ def instance_type_info(self) -> InstanceTypeInfo:
     @property
     def disable_simultaneous_multithreading_via_cpu_options(self) -> bool:
         """Return true if simultaneous multithreading must be disabled through cpu options."""
-        return self.disable_simultaneous_multithreading and self.instance_type_info.is_cpu_options_supported_in_lt()
+        return (
+            self.disable_simultaneous_multithreading
+            and self.instance_type_info.default_threads_per_core() > 1
```

Comment:
If this is not a strong concern for you, I think we should keep this conditions as it is because it guarantees that we don't do any unnecessary setup action, whether it is manual or via template.

Furthermore, since sooner or later we are moving towards a scenario where we will perform this specific disablement only manually, it would be even better to skip manual steps when not required.

What do you think?

Reply:
ACK